src/platform/library/include/platform/lib/memory.h

```diff
@@ -29,6 +29,10 @@
 
 #define SHARED_DATA
 
+#define uncache_to_cache(address)	address
+#define cache_to_uncache(address)	address
```

Comment:
I think address should be in `(address)`, it's weird that checkpatch didn't complain, it can lead to error when address is expression.